### PR TITLE
Fix/graphql resolvers and schemas

### PR DESCRIPTION
**Pull request by ResuBaka**
Here we had multible problems.

1. One was that some the like filter did not allow for multible strings as parameter.  
2. You could not filter/search for url_path of products
3. cms_blocks did not return the id so cms_blocks where not working in the frontend.
4. _source_include or  _source_exclude did not work anymore. This I have fixed with using the newer elasticsearch libary for es5

I have not tested it with the default theme and default data from the api but with our data it works fine.

### CODE REVIEW

**Comment by ResuBaka** [comment]
Which person should look over this PR? 
@lukeromanowicz

**Comment by pkarw** [comment]
@ResuBaka sorry for not responding sooner! @andrzejewsky will take care of this one!

File: src/graphql/elasticsearch/attribute/resolver.js [hunk 1]
```diff
@@ -3,21 +3,21 @@ import client from '../client';
 import { buildQuery } from '../queryBuilder';
 import { getIndexName } from '../mapping'
 
-async function listAttributes (attributes, context, rootValue, _sourceInclude) {
+async function listAttributes (attributes, context, rootValue, _source_include) {
   let query = buildQuery({ filter: attributes, pageSize: 150, type: 'attribute' });
 
-  if (_sourceInclude === undefined) {
-    _sourceInclude = config.entities.attribute.includeFields
+  if (_source_include === undefined) {
```
Comment (andrzejewsky) [inline src/graphql/elasticsearch/attribute/resolver.js]: `typeof _source_include === undefined` or just `if (!_source_include)` is enough

Comment (ResuBaka) [reply src/graphql/elasticsearch/attribute/resolver.js]: Changed it.

**Pull request merged**